### ISSUE TITLE
Is that possible to show time range options ?

### ISSUE THREAD

**Issue by shivam5x** [chuyik/jquery-datepicker-ios-android-theme#1]
Hi,
  I am using your theme in my app, now in my new task i need to show only breakfast time and lunch time, so is it possible to get time like from 7:00 am 10 : 00 am (predefined time range).

Thanks
Shivam

**Comment by chuyik** [comment]
Yes I can offer an option like "hourRange", but I found it a little bit complicated to implement as I try to develop, since the UI is handled differently in am.pm/24-hour format, and the date value is somehow related to the index of "hour" list.

But I've made some progress, so I think this request can be fulfilled in a few days.
